rdrf/registry/groups/models.py

```diff
@@ -96,11 +95,26 @@ def get_by_user(self, staff_user):
             return self.all()
 
         if staff_user.is_staff:
+
+            or_filters = [Q(working_groups__in=staff_user.working_groups.all()),
+                          Q(working_groups__isnull=True)]
+
+            if staff_user.is_clinician:
+                # Clinicians have special logic that potentially grants them access to users
+                # outside their working groups
+                from registry.patients.models import Patient
```

Comment:
local import

Reply:
It has to be local this time, otherwise it causes a circular dependency.